src/controllers/controller.cpp

```diff
@@ -85,6 +85,7 @@ void Controller::send(QList<int> data, unsigned int length) {
     // If you change this implementation, also change it in HidController (That
     // function is required due to HID devices having report IDs)
 
+    length = data.size();
```

Comment:
I don't think the `length` parameter should ever be different from `data.size()`. It would be an easy mistake pass the wrong length and confuse PortMidi or another backend.